models/arbitrum/arbitrum/arbitrum_arbitrum_airdrop_claims.sql

```diff
@@ -31,21 +32,21 @@ SELECT 'arbitrum' AS blockchain
 , t.recipient
 , t.contract_address
 , t.evt_tx_hash AS tx_hash
-, CAST(t.amount AS DECIMAL(38,0)) AS amount_raw
+, t.amount as amount_raw
 , CAST(t.amount/POWER(10, 18) AS double) AS amount_original
 , CASE WHEN t.evt_block_time >= (SELECT minute FROM early_price) THEN CAST(pu.price*t.amount/POWER(10, 18) AS double)
     ELSE CAST((SELECT price FROM early_price)*t.amount/POWER(10, 18) AS double)
     END AS amount_usd
-, '{{arb_token_address}}' AS token_address
+, {{arb_token_address}} AS token_address
 , 'ARB' AS token_symbol
 , t.evt_index
 FROM {{ source('arbitrum_arbitrum', 'TokenDistributor_evt_HasClaimed') }} t
 LEFT JOIN {{ ref('prices_usd_forward_fill') }} pu ON pu.blockchain = 'arbitrum'
-    AND pu.contract_address='{{arb_token_address}}'
+    AND pu.contract_address= {{arb_token_address}}
     AND pu.minute=date_trunc('minute', t.evt_block_time)
     {% if is_incremental() %}
-    AND pu.minute >= date_trunc("day", now() - interval '1 week')
+    AND pu.minute >= date_trunc('day', now() - interval '7' Day)
     {% endif %}
 {% if is_incremental() %}
-WHERE t.evt_block_time >= date_trunc("day", now() - interval '1 week')
+WHERE t.evt_block_time >= date_trunc('day', now() - interval '7' Day)
```

Comment:
The airdrop is only live for the next ~20 days so let's try to remember and remove the incremental run then - Source: https://arbitrum.foundation/